src/pybind/mgr/dashboard/frontend/src/app/shared/models/cd-table-action.ts

```diff
@@ -18,7 +18,7 @@ export class CdTableAction {
 
   // The font awesome icon that will be used
   icon: string;
-
+  title?: string;
```

Comment:
nice! :partying_face: let's add a comment here saying what this attribute is like we have for others